l10n_nl_intrastat/__openerp__.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Intrastat report for the Netherlands (ICP)
+#
+#    Based on lp:new-report-intrastat,
+#    Copyright (C) 2010-2011 Akretion (http://www.akretion.com).
+#    All Rights Reserved
+#    @author Alexis de Lattre <alexis.delattre@akretion.com>
+#
+#    Modifications Copyright (C) 2012 Therp BV <http://therp.nl>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```

Comment:
use short license headers: https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py#L1-L3, but keep the dates and people involved